src/core/appgateway.tf

```
@@ -413,6 +444,13 @@ module "app_gw" {
       priority              = 80
     }
 
+    selfcare-io-pagopa-it = {
+      listener              = "selfcare-io-pagopa-it"
+      backend               = "selfcare-io-app"
+      rewrite_rule_set_name = "rewrite-rule-set-selfcare-io"
+      priority              = 100
```

```suggestion
      priority              = 110
```

other PR used 100 value